Livecheckables/sdl.rb

```diff
@@ -0,0 +1,6 @@
+class Sdl
+  livecheck do
+    url "http://hg.libsdl.org/SDL/tags"
+    regex(/release-v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    url "https://www.libsdl.org/release/"
    regex(/href=.*?SDL-v?(1(?:\.\d+)+)\.t/i)
```

I would prefer to stick to the first-party release index, since it's where the stable archive is found. The `sdl` formula is only concerned with `1.x` releases (`2.x` releases are found in the `sdl2` formula), so we just have to make sure to only match appropriate versions.

There are two ways to accomplish this in the context of `sdl`. The file names consistently use `SDL-x.x.x` to refer to `1.x` releases and `SDL2-x.x.x` to refer to `2.x` releases. By only matching `SDL-` in the regex, we should only get `1.x` releases.

However, we can go also further and restrict matching to only versions with a major version of `1`, just to be sure (in case someone messes up and uses `SDL-` when they meant `SDL2-`). This is how we usually restrict matching for versioned formulae when the software name in the file name is the same for all releases and the only distinction is the major version.